libcodechecker/analyze/analyzer.py

```diff
@@ -231,8 +231,19 @@ def perform_analysis(args, context, actions, metadata):
     metadata['timestamps'] = {'begin': start_time,
                               'end': end_time}
 
-    if skip_handler:
-        metadata['skip_file_lines'] = skip_handler.skip_file_lines
+    # Remove previous skip file if there was any.
+    skip_file_to_send = os.path.join(args.output_path, 'skip_file')
+
+    if os.path.exists(skip_file_to_send):
+        os.remove(skip_file_to_send)
+
+    try:
+        if args.skipfile:
```

Comment:
Couldn't we use the following form to check if skipfile is given:
```py
if 'skipfile' in args:
    ...
```
This way the try/catch block can be removed.